src/session_pool/session.js

```diff
@@ -1,14 +1,48 @@
 import { cryptoRandomObjectId } from 'apify-shared/utilities';
 import log from 'apify-shared/log';
 import { checkParamOrThrow } from 'apify-client/build/utils';
-import tough from 'tough-cookie';
-import EVENTS from './events';
+import { Cookie, CookieJar } from 'tough-cookie';
+import { Cookie as PuppeteerCookie, SameSiteSetting } from 'puppeteer'; // eslint-disable-line import/named,no-unused-vars
+import { EventEmitter } from 'events'; // eslint-disable-line no-unused-vars
+import EVENTS from './events'; // eslint-disable-line import/named,no-unused-vars
 import { STATUS_CODES_BLOCKED } from '../constants';
 import { getCookiesFromResponse } from './session_utils';
 
-const { Cookie } = tough;
-
+/**
+ * Peristable {Session} state.
+ * @typedef {Object} SessionState
+ * @property {String} id
+ * @property {Object} cookies
+ * @property {CookieJar} cookieJar
```

Comment:
plain `Object`, produced by the `.toJSON()` call

Reply:
Technically, the `.toJSON()` call doesn't even need to be there, but it's still an `Object`

Reply:
Done.